quotes.json

```diff
@@ -2549,7 +2541,7 @@
     "text": "If you want to study yourself, look into the hearts of other people. If you want to study other people, look into your own heart."
   },
   {
-    "author": "Fyodor Dostoyevsky",
+    "author": "Fyodor Dostoevsky",
```

Comment:
Both are valid: https://en.wikipedia.org/wiki/Fyodor_Dostoevsky
But happy to use the more popular version. 👌